tests/terraform-eks/create/README.md

```diff
@@ -0,0 +1,43 @@
+## Terraform scripts to provision EKS cluster #
+***
+
+
+The Amazon Elastic Kubernetes Service (EKS) is the AWS service for deploying, managing, and scaling containerized applications with Kubernetes. This repo contain terraform files, to automate the EKS cluster deployment.
+***
+
+#### Requirements:
+- AWS account with the IAM permissions listed on the EKS module documentation
+- Configured AWS CLI
+- AWS IAM Authenticator
+- kubectl
+- terraform
+
+#### Deployment:
+- Initialize working directory:  
+``` terraform init```
+- Create execution plan:  
+``` terraform plan```
+- Create a cluster:  
+``` terraform apply```
+- Destroy the cluster:  
+``` terraform destroy```
+
+
+#### Files overview:
```

Comment:
there is main.tf not described in that list.

Reply:
Fixed